ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -4149,6 +4149,7 @@ interface profile TEST-PROFILE-2
    command ptp enable
 !
 ip virtual-router mac-address 00:1c:73:00:dc:01
+ip virtual-router mac-address advertisement-interval 40
 !
```

Comment:
Getting an extra `!` in cfg file

Running config output:
```
!
ip virtual-router mac-address 00:1c:73:00:dc:01
ip virtual-router mac-address advertisement-interval 40
ip address virtual source-nat vrf TEST_01 address 1.1.1.1
ip address virtual source-nat vrf TEST_02 address 1.1.1.2
ip address virtual source-nat vrf TEST_04 address 1.1.1.3
ip address virtual source-nat vrf Tenant_A_OP_Zone address 10.255.1.5
ipv6 address virtual source-nat vrf TEST_03 address 2001:db8:85a3::8a2e:370:7334
ipv6 address virtual source-nat vrf TEST_04 address 2001:db8:85a3::8a2e:370:7335
```

May be we can combine two templates ip-virtual-router-mac-address.j2 and virtual-source-nat-vrfs.j2 @guillaumeVilar @ClausHolbechArista ?

Reply:
ignore for now